pyproject.toml

```diff
@@ -47,7 +47,8 @@ classifiers = [
 [tool.poetry.dependencies]
 python = "^3.8"
 PyHamcrest = ">=2.0.0"
-typing-extensions = ">=4.1.1"
```

Comment:
Why remove this?  We use typing_extensions in the project.

Reply:
Hm... this might have been a mistake. I think i was having some trouble with one of the installs. I can't remember, it's been too long!